ENT-3136 | Added a new task for sending the offer usage emails.

hasnain-naveed opened the pull request.

irfanuddinahmad commented on ecommerce_worker/sailthru/v1/send_notification.py:
```diff
@@ -0,0 +1,133 @@
+
+class SendNotification(object):
+    """
+    This class exposing the 'send' function for sending the emails.
```
nit: exposing -> exports

hasnain-naveed commented:
I have written a generic new class for sending the notification/email which will send the single or multi_send. `send_offer_update_email` and `send_offer_assignment_email` will also use that class. But I didn't change those in this PR. I have added TODOs and also added a story for changing those functions.
